src/networks/DFP.ts

```diff
@@ -39,7 +39,11 @@ class DfpAd implements INetworkInstance {
     this.breakpoints = breakpoints;
 
     googletag.cmd.push(() => {
-      this.slot = googletag.defineSlot(path,  Array.isArray(sizes) ? sizes : [0, 0], this.id);
+      this.slot = googletag.defineSlot(path, Array.isArray(sizes) ? sizes : [0, 0], this.id);
+
+      if (collapsible) {
+        this.slot.setCollapseEmptyDiv(true, true);
```

Comment:
is there a difference between `this.slot.setCollapseEmptyDiv(true, true);` and `this.slot.setCollapseEmptyDiv(true);`?

Reply:
Yes.
```
setCollapseEmptyDiv() = do not allow collapse
setCollapseEmptyDiv(true) = allow ad to collapse only after response does not return an ad
setCollapseEmptyDiv(true, true) = Begin collapsed and only expand if ad is populated
```

Reply:
For some reason, the method was not working properly with the, most likely preferred, second case shown above.